### ISSUE TITLE
Dump only specific files

### ISSUE THREAD

**Issue by Symbai** [Wunkolo/UWPDumper#19]
Either by exact name: `processID -filter bla.dll`

or (if possible) by wildcard like: `processID -filter *.dll`

This avoids copying a lot of unnecessary files which then is also a lot faster.

**Comment by Wunkolo** [comment]
Sounds like a good idea. Though at the moment there isn't a command-line-only way to run the dumper.
Passing in a regex expression to test the file paths against, and having the default be `.+`, sounds like a great addition though.

**Comment by Wunkolo** [comment]
There is now a command-line-only way to run UWPInjector since this issue has been created and this might be much more possible now.
The design would be `-f` or `-filter` to pass a regex expression over to the injected process so that it only dumps files matching the target expression.

**Comment by sunbeam906** [comment]
Could you kindly provide an example on how these regular expressions should look like?

I am asking because I've tried in every single possible way to dump "Starfield.exe" and UWP Dumper still dumps whatever it likes. If I give it "Starfield.*exe" it dumps any file that has "Starfield" in its naming convention, disregarding the ".*exe" part completely. If I give it "exe", it dumps any file. With or without the "".

Thank you,
Sun

**Comment by Symbai** [comment]
Its not implemented yet. He only spoke about a possible design / solution.

**Comment by namish5502** [comment]
Yes, you can do that in one line by editing it from where the dumping will start, but, yes or yes, it has to be with the application where you will get the files from, otherwise it will not work.
example:

cuphead = base game "installs in another path"

DLC = "which is installed in a different path" 

To do this you just have to edit this line where the dumping start route will begin and with that you are left to be able to dump what you want or something like that it would be called 

But yes or if you have to run that application and the file to be extracted is part of it in terms of the ID that it launches when it is executed and the binary

for( auto& Entry : fs::recursive_directory_iterator(UWP::Current::GetPackagePath()) )
